### PR TITLE
Fix Wrong srec_info output when using the last 32-bit address

### PR DESCRIPTION
**Pull request by jtxa**
This should fix all issues from #10

### CODE REVIEW

File: srec_info/main.cc [hunk 1]
```diff
@@ -17,10 +17,12 @@
...
 //
 
+#include <cstdio>
+#include <cstdint>
+#include <cstdlib>
 #include <iomanip>
 #include <iostream>
-#include <cstdlib>
-#include <cstdio>
+#include <limits>
```
Comment (jtxa) [inline srec_info/main.cc:25]: Useless include. I wanted to use `numeric_limits` but it didn't worked for `uint32_t`.
The limit is now hardcoded as double.

File: srec_info/main.cc [hunk 2]
```diff
@@ -151,34 +156,47 @@ main(int argc, char **argv)
...
-            std::cout << "   Holes: " << buf << "%" << std::endl;
+            std::cout << "Filled: ";
+            if (number_bytes == 0UL)
+                std::cout << "100000000";
+            else
+                std::cout << std::setw(prec) << number_bytes;
+            std::cout << std::endl;
+            const uint32_t range_size = static_cast<uint32_t>(range_highest - range_lowest);
+            const double real_number_bytes = (number_bytes == 0UL) ? 4294967295.0 : number_bytes;
+            const double real_range_size = (range_size == 0UL) ? 4294967295.0 : range_size;
```
Comment (jtxa) [inline srec_info/main.cc:189]: That's wrong, i wanted to use 4294967296.0 (max+1)

**Comment by sierrafoxtrot** [comment]
CI workflow has been updated in master to remove the failing test report step. Merging should let this pass.

**Comment by jtxa** [comment]
rebased to current master

**Pull request merged**